.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 node_modules
+package-lock.json
```

Comment:
why are you removing the lock file and ignoring it? I think this is redundant

Reply:
package-lock.json is used for development, but when publishing a package to npm it is still ignored. Despite the npm documentation, I usually remove it anyway, in order to always have fresh dependencies. If you think it's redundant, then I can return it.

Reply:
I usually leave the local file, but for projects that have a more complex structure. OK, let's leave your option